components/formats-gpl/src/loci/formats/in/SlideBook6Reader.java

```diff
@@ -0,0 +1,402 @@
+		// invalid (less than 0 or >= the number of images). The ordering of the array returned by openBytes should
+		// correspond to the values returned by isLittleEndian() and isInterleaved(). Also, the length of the byte array
+		// should be [image width * image height * bytes per pixel]. Extra bytes will generally be truncated. It is
+		// recommended that the first line of this method be:
+		//    FormatTools.checkPlaneParameters(this, no, buf.length, x, y, w, h) - this ensures that all of the parameters
+		// are valid.
+
+		FormatTools.checkPlaneParameters(this, no, buf.length, x, y, w, h);
+
+		int[] zct = FormatTools.getZCTCoords(this, no);
+		int bpc = FormatTools.getBytesPerPixel(getPixelType());
+		byte[] b = new byte[FormatTools.getPlaneSize(this)];
+
+		readImagePlaneBuf(b, getSeries(), 0, zct[2], zct[0], zct[1]);
+
+		int pixel = bpc * getRGBChannelCount();
+		int rowLen = w * pixel;
+		for (int row=0; row<h; row++) {
+			System.arraycopy(b, pixel * ((row + y) * getSizeX() + x), buf,
+					row * rowLen, rowLen);
+		}
+
+		if (isRGB()) {
+			int bpp = getSizeC() * bpc;
+			int line = w * bpp;
+			for (int row=0; row<h; row++) {
+				for (int col=0; col<w; col++) {
+					int base = row * line + col * bpp;
+					for (int bb=0; bb<bpc; bb++) {
+						byte blue = buf[base + bpc*(getSizeC() - 1) + bb];
+						buf[base + bpc*(getSizeC() - 1) + bb] = buf[base + bb];
+						buf[base + bb] = blue;
+					}
+				}
+			}
+		}
+		return buf;
+	}
+
+	// -- Internal FormatReader API methods --
+	public void close(boolean fileOnly) throws IOException {
+		// Cleans up any resources used by the reader. Global variables should be reset to their initial state, and any
+		// open files or delegate readers should be closed.
+		super.close(fileOnly);
+
+		// Cleans up any resources used by the reader. Global variables should be reset to their initial state, and any
+		// open files or delegate readers should be closed.
+		closeFile();
+	}
+
+	/* @see loci.formats.FormatReader#initFile(String) */
+	protected void initFile(String id) throws FormatException, IOException {
+		super.initFile(id);
+
+		// The majority of the file parsing logic should be placed in this method. The idea is to call this method
+		// once (and only once!) when the file is first opened. Generally, you will want to start by calling
+		// super.initFile(String). You will also need to set up the stream for reading the file, as well as initializing
+		// any dimension information and metadata. Most of this logic is up to you; however, you should populate the 'core'
+		// variable (see loci.formats.CoreMetadata).
+
+		// Note that each variable is initialized to 0 or null when super.initFile(String) is called. Also,
+		// super.initFile(String) constructs a Hashtable called "metadata" where you should store any relevant metadata.
+
+		try {
+			openFile(id);
+			int numSeries = getNumCaptures();
```

Comment:
Is it possible to have multiple positions or montage tiles per capture?  If so, ```numSeries``` should likely reflect the sum of all positions and montage tiles across all captures - otherwise only the first position from each tile will be accessible.